PROTOTYPE_NOTES.md

```diff
@@ -0,0 +1,130 @@
+# Subgraph Batching Prototype
+Version: Draft 0.1
+
+These notes are designed to maximise the utility of evaluating the subgraph batching prototype both for the customers involved and for Apollo.
+
+Please read these notes carefully before:
+ - Using the prototype router 
+ - Reporting results back to Apollo
+
+The [subgraph batching design document](https://docs.google.com/document/d/1KlGgNCm1sQWc-tYs2oqHauMBIYdrtlpIELySZDAnISE/edit?usp=sharing) may be helpful in evaluating the prototype.
+
+## Using the prototype
+
+### Deployment
+
+#### Accessing the prototype
+
+You can choose to build from source or from one of our pre-built options.
+
+##### Source
+
+The prototype router is publicly available on the main router repo on branch `subgraph-batching-prototype`. It can be accessed and built as normally.
```

Comment:
```suggestion
The prototype router is publicly available on the main router repo on branch `subgraph-batching-prototype`. It can be accessed and built as normal.
```